controller/src/main/java/org/jboss/as/controller/services/path/PathResourceDefinition.java

```diff
@@ -134,6 +133,7 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
         SimpleAttributeDefinition pathAttr = specified ? PATH_SPECIFIED : PATH_NAMED;
         resourceRegistration.registerReadWriteAttribute(pathAttr, null, new PathWriteAttributeHandler(pathManager, pathAttr, services));
         resourceRegistration.registerReadOnlyAttribute(READ_ONLY, null);
+        PathSizeMetricHandler.registerMetric(resourceRegistration, pathManager, PATH, RELATIVE_TO);
```

Comment:
I wonder if this should be a custom operation instead of a metric. The file walking can be expensive if a path points to the root of a large tree, so calculating it when people/tools do common things like :read-resource(include-runtime=true) can have unintended effects.